### ISSUE TITLE
Support for multiple input files with goby log convert

### ISSUE THREAD

**Issue by jason-jaia** [GobySoft/goby3#326]
I would like to be able to run goby log tool like so:  "goby log convert *.goby --format=HDF5"

**Comment by tsaubergine** [comment]
The second (optional) command line positional parameter is "outfile":
```
goby log convert <input_file> [<output_file>] [options]
```
so there's no way to know the difference between multiple infiles and a specified outfile. I guess we could check for the .goby extension, but it seems trivial to just use a loop

```
for file in *.goby; do goby log convert $file --format=HDF5; done
```

**Comment by jason-jaia** [comment]
I agree it's trivial to script, but I guess I'm lazy. I also see your point that it breaks the structure of the command, so it seems like the effort/reward balance probably doesn't make sense.

**Comment by SBFRF** [comment]
for clarity, this for loop is `bash`

**Comment by tsaubergine** [comment]
I see the justification for this more when we can do multiple parallel conversions. This will be included in 3.2.0. See #333.
